integration-cli/docker_cli_run_unix_test.go

```diff
@@ -299,7 +299,10 @@ func (s *DockerSuite) TestRunEchoStdoutWithMemoryLimit(c *check.C) {
 	}
 }
 
-// should run without memory swap
+// TestRunWithoutMemoryswapLimit should run without memory swap limit.
```

Comment:
I don't get why this comment is needed since the test name is explicitly telling you that you'll run a container w/o memory swap limit